zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/util/SqlSplitter.java

```diff
@@ -79,11 +81,9 @@ public SqlSplitter(String... additionalSingleCommentPrefixList) {
       }
 
       // end of multiple line comment
-      if (multiLineComment && character == '/' && text.charAt(index - 1) == '*') {
+      if (multiLineComment && (index - 1) >= 0 && text.charAt(index - 1) == '/'
```

Comment:
Can we replace ` (index - 1) >= 0` with ` (index - 2) >= 0` and just skip it later?

Reply:
Sorry, I don't get it. Here I just check whether it is the end of multiple comment via checking whether (index-1) is `/` and (index-2) is `*`